### ISSUE TITLE
Creating a Custom Policy - add info about max number of Condition Values

### ISSUE THREAD

**Issue by dsabelni** [opentelekomcloud-docs/identity-access-management#53]
https://github.com/opentelekomcloud-docs/identity-access-management/blob/main/umn/source/user_guide/permissions/creating_a_custom_policy.rst - please add information to the page about the maximum possible number of Values in the Request Condition is 10. For example, in the Table 1 Condition parameters to the Description column of the Value row:

> Used together with a condition key and an operator that requires a keyword, to form a complete condition statement. Maximum number of values - 10.

**Comment by lbelejka** [comment]
A key in the Condition element of a statement. There are global and service-level condition keys. Global condition keys (starting with g:) are available for operations of all services, while service-level condition keys (starting with a service abbreviation name such as obs:) are available only for operations of the corresponding service.

**Comment by dsabelni** [comment]
@lbelejka can you please explain your comment? The "Condition Key" description doesn't say anything about the limitations of the Specific Conditions values number.

I've tried creating a policy with a condition that has 11 values and it worked. To be honest I don't recall the initial reason or scenario behind this ticket,  so I can't reproduce the original issue. @lbelejka can you please confirm that there is no limit on Specific Conditions values?

**Comment by lbelejka** [comment]
> [@lbelejka](https://github.com/lbelejka) can you please explain your comment? The "Condition Key" description doesn't say anything about the limitations of the Specific Conditions values number.
> 
> I've tried creating a policy with a condition that has 11 values and it worked. To be honest I don't recall the initial reason or scenario behind this ticket, so I can't reproduce the original issue. [@lbelejka](https://github.com/lbelejka) can you please confirm that there is no limit on Specific Conditions values?

@dsabelni, yes, the limit has been lifted, but we have to wait for the Doc update. I overlooked that it was UM, so I will contact our UAT. Sorry for confusing.

**Comment by dsabelni** [comment]
OK, thank you!
